Os/test/ut/rawtime/RawTimeRules.cpp

```diff
@@ -38,10 +38,16 @@ bool Os::Test::RawTime::Tester::Now::precondition(const Os::Test::RawTime::Teste
 
 void Os::Test::RawTime::Tester::Now::action(Os::Test::RawTime::Tester& state) {
     FwIndexType index = state.pick_random_index();
+    Os::RawTime& raw_time_under_test = state.m_times[index];
 
-    Os::RawTime::Status status = state.m_times[index].now();
-    state.shadow_now(index);
+    auto lower_time = std::chrono::system_clock::now();
+    Os::RawTime::Status status = raw_time_under_test.now();
+    auto upper_time = std::chrono::system_clock::now();
```

Comment:
I suspect `std::chrono::system_clock::now()` to potentially not behave as expected in this test on some platforms (some would give time-since-boot). But then again, Os::RawTime may give the same values? Thoughts?
I did note that these tests are intended to be portable, but some restrictions apply... so some platforms can choose to implement tests on their own.

Reply:
I think this is fine